package.json

```diff
@@ -461,6 +468,7 @@
     "depcheck": "^1.4.3",
     "dependency-tree": "^10.0.9",
     "detect-port": "^1.5.1",
+    "dotenv": "^16.3.1",
```

Comment:
Question: Could we have leveraged `.metamaskrc` to load environment variables instead of introducing `dotenv`?

Reply:
I believe the .metamaskrc file is designated for MetaMask-related configurations, whereas the .env file is utilised for end-to-end Playwright testing. My preference is to maintain these configurations separately for clarity and organisation.

Reply:
💯